File: api/v2010/member/update-example-1/update-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$member = $twilio->queues("QU5ef8732a3c49700934481addd5ce1659")
                 ->members("Front")
                 ->update("POST", "http://demo.twilio.com/docs/voice.xml");

print($member.callSid);

File: api/v2010/member/update-example-1/update-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Queue;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var member = MemberResource.Update(
            url: new Uri("http://demo.twilio.com/docs/voice.xml"),
            method: Twilio.Http.HttpMethod.Post,
            pathQueueSid: "QU5ef8732a3c49700934481addd5ce1659",
            pathCallSid: "Front"
        );

        Console.WriteLine(member.CallSid);
    }
}
